src/allmydata/test/test_storage.py

```diff
@@ -646,41 +648,42 @@ def test_seek(self):
         f2 = open(filename, "rb")
         self.failUnlessEqual(f2.read(5), b"start")
 
-
-    def test_leases(self):
-        ss = self.create("test_leases")
+    def create_bucket(self, ss, storage_index, expected_already=0, expected_writers=5):
+        """
+        Given a StorageServer, create a bucket and return renewal and
+        cancellation secrets.
+        """
         canary = FakeCanary()
         sharenums = list(range(5))
```

Comment:
Having the 5 in here and the two expected values as parameters is a little awkward.  I'd at least document this as trying to create 5 shares.